src/modules/blog/entities/blog-comment.entity.ts

```diff
@@ -0,0 +1,17 @@
+import { Entity, PrimaryGeneratedColumn, Column, ManyToOne } from 'typeorm';
+import { Blog } from './blog.entity';
+import { AbstractBaseEntity } from '../../../entities/base.entity';
+import { User } from '../../user/entities/user.entity';
+
+@Entity()
+export class BlogComment extends AbstractBaseEntity {
+  @PrimaryGeneratedColumn()
+  @Column('text', { nullable: false })
```

Comment:
You don't need @PrimaryGeneratedColumn() since you're extending AbstractBaseEntity